src/common/notifications.cpp

```diff
@@ -315,3 +319,99 @@ void notifications::show_toast_with_activations(std::wstring message, std::wstri
 
     notifier.Show(notification);
 }
+
+void notifications::show_toast_with_progress_bar(std::wstring message, const std::wstring& version, toast_params params)
```

Comment:
There's a lot of copypaste in this function from `show_toast_with_activations`. Let's merge them together into `show_toast_with_activations`

Reply:
Done